package.json

```diff
@@ -64,6 +65,10 @@
     "core-js": "^2.0.0",
     "normalize.css": "^4.0.0",
     "react": "^15.0.0",
-    "react-dom": "^15.0.0"
+    "react-dom": "^15.0.0",
+    "react-redux": "^5.0.4",
+    "react-router-dom": "^4.1.1",
+    "redux": "^3.6.0",
+    "styled-components": "^1.4.6"
```

Comment:
Didn't use `styled-components` but don't think it will work in long run... I think better way is to use `namespaces` in stylus and some loader to load it automatically (if you don't want to import them implicit [the stylus files] ) 

for example  
```stylus
// MyA.styl
#ns(my-a)
   color #ddd
   .title
      font-size 16px
```
```jsx
// MyA.jsx
import namespaced from 'namespaced'; // https://github.com/hvesalai/namespaced

const cln = namespaced('my-a');

const MyA = ({children}) => (
  <div className={cln(':')}>
     <div className={cln(':title')}>MyA</div>
     {children}
  </div>
);

export default MyA;

```
IMHO this more clean, easy to read and maintain,